packages/calling-component-bindings/src/callControlSelectors.ts

```diff
@@ -168,6 +170,36 @@ export const raiseHandButtonSelector: RaiseHandButtonSelector = reselect.createS
   }
 );
 
+/* @conditional-compile-remove(reaction) */
+/**
+ * Selector type for {@link ReactionButton} component.
+ *
+ * @beta
+ */
+export type ReactionButtonSelector = (
+  state: CallClientState,
+  props: CallingBaseSelectorProps
+) => {
+  checked?: boolean;
+  disabled?: boolean;
+};
+
+/* @conditional-compile-remove(reaction) */
+/**
+ * Selector for {@link ReactionButton} component.
+ *
+ * @beta
+ */
+export const reactionButtonSelector: ReactionButtonSelector = reselect.createSelector(
+  [getLocalParticipantReactionState, getCallState],
+  (reaction, callState) => {
+    return {
+      checked: reaction ? true : false,
+      disabled: callState === 'Connected' ? true : false
```

Comment:
reverse this, when it's connected it should be false (i.e. not disabled)

```suggestion
      disabled: callState !== 'Connected'
```

Reply:
I dont think the selector implementation is taking effect. Because this is clearly wrong logic and and it still worked.

Reply:
Thanks. Updating this.